README.md

```diff
@@ -43,12 +43,16 @@ then the workspace looks like below
 ```
 $ cd <workspace>/build_media
 ```
-6. 
+6.  Export the current LIBVA installation directory(ies)  
 ```
-$ cmake ../media-driver
+$  export LIBVA_DRIVERS_PATH=/usr/lib/x86_64-linux-gnu/dri:/usr/lib/dri
 ```
 7. 
 ```
+$ cmake ../media-driver -D[options]
```

Comment:
I don't see the point of adding `-D[options]`, that's not a cmake documentation and this makes command example not copy-pastable.

Reply:
Was attempting to bring in similarities to https://github.com/intel/gmmlib#building